### DISCUSSION TITLE
How to set different background color in no audio area?

### DISCUSSION THREAD

**Discussion by TD-jizhijun** [katspaugh/wavesurfer.js#3863] [Q&A] [answered]
I have a design here. If there is no voice, I need to add a different color mark in this area. Is there any way to achieve this effect? ​​Thank you!

<img width="607" alt="image" src="https://github.com/user-attachments/assets/229b6239-c0db-43a0-8a4b-d7d7158046c1">

**Comment by katspaugh** [comment, marked as answer]
You can use the Regions plugin for that: https://wavesurfer.xyz/examples/?silence.js
